client/src/components/continuousLegend/index.js

```diff
@@ -115,7 +116,13 @@ class ContinuousLegend extends React.Component {
       d3.select("#continuous_legend").selectAll("*").remove();
     }
 
-    if (colorAccessor && colorScale && colorScale.range) {
+    if (
```

Comment:
this would be a lot more readable if it was something like:
```
const range = colorScale?.range;
const [domainMin, domainMax] = colorScale?.domain?.() ?? [0, 0];
if (colorAccessor && colorScale && range && domainMin < domainMax) {
  ...
}
```

and then use those variables further down